recipes/bilby/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "bilby" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c97db3251e2bc451bc9c6992e39d30f29f7646733547b000483bef18917e89b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  entry_points:
+    - bilby_plot=cli_bilby.plot_multiple_posteriors:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - future
+    - dynesty
+    - corner
+    - numpy >=1.9
+    - matplotlib >=2.0
+    - deepdish
+    - pandas
+    - scipy
+
+test:
+  commands:
+    - setx MPLBACKEND "Agg"  # [win]
+    - export MPLBACKEND="Agg"  # [unix]
+    - bilby_plot --help
+    - python -c "import bilby"
+    - python -c "import bilby.core"
+    - python -c "import bilby.gw"
+    - python -c "import bilby.hyper"
+    - python -c "import cli_bilby"
+
+about:
+  home: https://git.ligo.org/lscsoft/bilby
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
```

Comment:
It would be good to make a PR upstream to add the license file to the pypi upload.

Reply:
@beckermr, see https://git.ligo.org/lscsoft/bilby/merge_requests/323